### PR TITLE
docs: update custom echarts options sample

### PR DESCRIPTION
**Pull request by plagoa**
- Allow the use of `MarkLineComponent` on the `LineChart` and `BarChart` components
- Update the custom echart options sample for the `LineChart` component

### CODE REVIEW

**Review by HQFOX — APPROVED** [review]

File: packages/viz/src/hooks/useSeries.tsx [hunk 1]
```diff
@@ -174,15 +175,15 @@ export const useSeries = ({
...
     measures,
     type,
-    horizontal,
+    nameFormatter,
     radius,
+    stack,
+    horizontal,
+    emptyCellMode,
+    area,
+    areaOpacity,
```
Comment (zettca) [inline packages/viz/src/hooks/useSeries.tsx:186]: I'm confused; nothing changed here?

File: packages/viz/src/hooks/useSeries.tsx [hunk 2]
```diff
@@ -52,7 +52,8 @@ export const useSeries = ({
   areaOpacity = 0.5,
   emptyCellMode,
   radius,
-}: HvSeriesHookProps) => {
+}: // markLine,
```
Comment (zettca) [inline packages/viz/src/hooks/useSeries.tsx:55]: is this supposed to be included?

File: packages/viz/src/LineChart/LineChart.stories.tsx [hunk 3]
```diff
@@ -798,9 +798,28 @@ export const CustomEchartsOptions: StoryObj<HvLineChartProps> = {
...
         onOptionChange={(option) => {
+          console.log(option);
           if (Array.isArray(option.yAxis) && option.yAxis.length === 1) {
             option.yAxis = [{ ...option.yAxis[0], splitNumber: 8 }];
           }
+          option.series[0].markLine = {
+            data: [{ yAxis: 3000, name: "Average" }],
+            symbol: "none",
+            itemStyle: {
+              color: "#333333",
```
Comment (zettca) [inline packages/viz/src/LineChart/LineChart.stories.tsx:809]: This isn't visible in wicked 😬
image


Can't we use colors from the palette? If not, we could use a non-gray color (eg. orange?)

File: packages/viz/src/hooks/useSeries.tsx (hunk 2 above)
Comment (plagoa) [reply packages/viz/src/hooks/useSeries.tsx]: Fixed

**Comment by plagoa** [comment]
@zettca updated the sample to use our theme colors.

**Review by zettca — APPROVED** [review]
Nice 👍

**Pull request merged**